assets/wire/utils.ts

```diff
@@ -214,10 +214,11 @@ function splitWords(str: string): Array<string> {
         .replace(/ {2,}/g, ' ')
 
         .trim()
-        .split(' ');
+        .split(/(?= [A-Z])/);
```

Comment:
can you add a comment what is it supposed to do? and not sure if this will work with french eg. `ÉQUILIBRE`